ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/GeoCityLookup.java

```diff
@@ -92,7 +109,7 @@ public PubsubMessage apply(PubsubMessage value) {
     }
   }
 
-  private final Fn fn = new Fn();
+  private  Fn fn = new Fn();
```

Comment:
why not final? also nit: `s/(private ) /\1/`

Reply:
This change was unintentional. Will revert.